### PR TITLE
Replace akka-http-core-experimental with akka-http-core

### PR DESCRIPTION
**Pull request by tkqubo**
This PR will update akka libraries, to use akka-pusher with akka-http.
⚠️ after this change, spray can no longer be supported, as it doesn't run with akka-http 2.4.x

### CODE REVIEW

**Comment by dtaniwaki** [comment]
@tkqubo Thank you for the PR! Could you remove unnecessary version change such as Scalia version and squash the commits?
I will do it outside this PR if necessary.

**Comment by tkqubo** [comment]
OK I removed unnecessary lines and tidied up commits :-)

And one thing I want to inform you that since Akka 2.4.0 dropped the support of Scala 2.10.x, I removed relevant lines.

File: src/main/scala/com/github/dtaniwaki/akka_pusher/PusherActor.scala [hunk 1]
```diff
@@ -23,7 +23,16 @@ class PusherActor(
   }
 
   implicit val system = context.system
-  implicit val ec: ExecutionContext = system.dispatcher
```
Comment (dtaniwaki) [inline src/main/scala/com/github/dtaniwaki/akka_pusher/PusherActor.scala:26]: Is this change necessary to update Akka version?

File: src/test/resources/application.conf [hunk 2]
```diff
@@ -7,4 +7,14 @@ pusher {
   secret=${?PUSHER_API_SECRET}
   batchTrigger = false
   batchInterval = 1000
+  dispatcher {
```
Comment (dtaniwaki) [inline src/test/resources/application.conf:10]: ditto

**Review by dtaniwaki — CHANGES_REQUESTED** [review]
@tkqubo Please check the comments and update the branch.

File: .travis.yml [hunk 3]
```diff
@@ -6,15 +6,13 @@ cache:
...
-  - 2.10.6
   - 2.11.7
 
 jdk:
   - oraclejdk8
 
 script:
-  - "[ \"$TRAVIS_SCALA_VERSION\" != \"2.10.6\" ] || sbt ++$TRAVIS_SCALA_VERSION clean test"
-  - "[ \"$TRAVIS_SCALA_VERSION\" == \"2.10.6\" ] || sbt ++$TRAVIS_SCALA_VERSION clean coverage test"
+  - "sbt ++$TRAVIS_SCALA_VERSION clean coverage test"
```
Comment (dtaniwaki) [inline .travis.yml:15]: Should be `clean test` if this change is for Scala 2.11.x?

**Review by dtaniwaki — CHANGES_REQUESTED** [review]

**Comment by dtaniwaki** [comment]
It seems your repository includes a lot of unnecessary changes. I will do the same change by myself. Thanks for the reporting!

**Comment by tkqubo** [comment]
Sorry for not responding long time.  I will leave this to you.  Thanks!

**Pull request closed without merging**